CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.4.0
```

Comment:
Given this is not realy a user-facing feature I'd opt for a patch release, 3.3.1